models/classes/LtiProvider/LtiProviderFactory.php

```diff
@@ -97,11 +98,17 @@ public function createFromArray(array $provider): LtiProvider
 
     private function getLtiVersion(array $propertiesValues): string
     {
-        if (empty($propertiesValues[RdfLtiProviderRepository::LTI_VERSION])) {
+        $ltiVersionResource = reset($propertiesValues[RdfLtiProviderRepository::LTI_VERSION]);
+
+        if ($ltiVersionResource instanceof core_kernel_classes_Literal && !empty(trim($ltiVersionResource->literal))) {
+            return $ltiVersionResource->literal;
```

Comment:
Are you sure we should return this and not compare with `RdfLtiProviderRepository::LTI_V_13`? (I am not :/)

Reply:
To be honest I do not know that this literal would hold the RDF representation or the LTI version number

Reply:
Agree. I'm not sure what I can expect here. I will compare this to expected string `1.3`.